webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -36,14 +33,3 @@ export const hideLoading = () => {
 export const changeActive = () => {
   deprecate('changeActive', '', '3.8');
```

Comment:
This reminds me: we need to drop deprecated code. Looks like this function was actually a noop and can also be dropped. Probably doesn't belong in this PR though.